src/main/java/com/assure/movie/dto/ActorDTO.java

```diff
@@ -2,8 +2,9 @@
 /**
  * @author Israel Yasis
  */
-public class ActorDTO implements DTO{
+public class ActorDTO implements DTO {
```

Comment:
Do we need to implement that DTO interface?

Reply:
The DTO's that I defined should extends from an empty **Abstract Class** or an **Interface** because they need to be part from an abstraction in order to be used in the converter pattern in a more easy way.

> public interface GenericConverter<E extends Identifiable, D extends DTO>

Reply:
Ok, I understand, thanks for the clarification.